### PR TITLE
Always Use .Release.Namespace in the ClusterRoleBinding Name

### PR DESCRIPTION
**Pull request by mandarjog**
Ensures that all clusterRoleBinding has name of namespace encoded in it.

This is needed for one-namespace testing.

### CODE REVIEW

**Comment by mandarjog** [comment]
fixes https://github.com/istio/istio/issues/4547

File: Makefile [hunk 1]
```diff
@@ -600,56 +600,15 @@ generate_yaml:
...
-		  --set global.rbacEnabled=true \
-		  --set istiotesting.oneNamespace=true \
-                  --set prometheus.enabled=true \
-				install/kubernetes/helm/istio > install/kubernetes/istio-one-namespace.yaml
-
 
-istio-auth.yaml: $(HELM)
+# creates istio.yaml istio-auth.yaml istio-one-namespace.yaml istio-one-namespace-auth.yaml
+# Ensure that values-$filename is present in install/kubernetes/helm/istio
+isti%.yaml: $(HELM)
```
Comment (cmluciano) [inline Makefile]: typo?

Comment (mandarjog) [reply Makefile]: `istio%.yaml` does not match `istio.yaml`.  `%` is one or more ?

Comment (cmluciano) [reply Makefile]: sorry missed the matching here

**Comment by mandarjog** [comment]
```
I0327 15:44:03.254] === RUN   TestDashboards/Mixer
I0327 15:44:05.228] 2018-03-27T15:44:05.228309Z	info	Running command kubectl get pods -n dashboard-test-ffab4487e02a4a4d9a6d09 -l app=echosrv -o jsonpath={.items[*].metadata.name}
I0327 15:44:05.552] 2018-03-27T15:44:05.552233Z	info	Command output: 
I0327 15:44:05.552] echosrv-deployment-77878768fb-xgqnw
I0327 15:44:05.553] 2018-03-27T15:44:05.552299Z	info	Running command kubectl exec -n dashboard-test-ffab4487e02a4a4d9a6d09 echosrv-deployment-77878768fb-xgqnw -c echosrv -- /usr/local/bin/fortio curl http://istio-telemetry.dashboard-test-ffab4487e02a4a4d9a6d09:9093/metrics
I0327 15:44:06.246] 2018-03-27T15:44:06.245995Z	info	Command output: 
I0327 15:44:06.246] 15:44:06 W http_client.go:584> Parsed non ok code 503 (HTTP/1.1 503)
I0327 15:44:06.246] 15:44:06 E commonflags.go:104> Error status 503 : HTTP/1.1 503 Service Unavailable\r\ncontent-length: 57\r\ncontent-type: text/plain\r\ndate: Tue, 27 Mar 2018 15:44:06 GMT\r\nserver: envoy\r\n\r\nupstream connect error or disconnect/reset before headers
I0327 15:44:06.247] HTTP/1.1 503 Service Unavailable
I0327 15:44:06.247] content-length: 57
I0327 15:44:06.247] content-type: text/plain
I0327 15:44:06.247] date: Tue, 27 Mar 2018 15:44:06 GMT
I0327 15:44:06.247] server: envoy
I0327 15:44:06.247] 
I0327 15:44:06.247] upstream connect error or disconnect/reset before headerscommand terminated with exit code 1
I0327 15:44:06.247] 2018-03-27T15:44:06.246066Z	info	Command error: exit status 1
I0327 15:44:06.247] === RUN   TestDashboards/Pilot
I0327 15:44:07.803] 2018-03-27T15:44:07.802783Z	info	Running command kubectl get pods -n dashboard-test-ffab4487e02a4a4d9a6d09 -l app=echosrv -o jsonpath={.items[*].metadata.name}
I0327 15:44:08.133] 2018-03-27T15:44:08.133253Z	info	Command output: 
I0327 15:44:08.133] echosrv-deployment-77878768fb-xgqnw
I0327 15:44:08.134] 2018-03-27T15:44:08.133333Z	info	Running command kubectl exec -n dashboard-test-ffab4487e02a4a4d9a6d09 echosrv-deployment-77878768fb-xgqnw -c echosrv -- /usr/local/bin/fortio curl http://istio-pilot.dashboard-test-ffab4487e02a4a4d9a6d09:9093/metrics
I0327 15:44:08.811] 2018-03-27T15:44:08.810993Z	info	Command output: 
I0327 15:44:08.812] 15:44:08 W http_client.go:584> Parsed non ok code 503 (HTTP/1.1 503)
I0327 15:44:08.812] 15:44:08 E commonflags.go:104> Error status 503 : HTTP/1.1 503 Service Unavailable\r\ncontent-length: 57\r\ncontent-type: text/plain\r\ndate: Tue, 27 Mar 2018 15:44:08 GMT\r\nserver: envoy\r\n\r\nupstream connect error or disconnect/reset before headers
I0327 15:44:08.812] HTTP/1.1 503 Service Unavailable
I0327 15:44:08.812] content-length: 57
I0327 15:44:08.812] content-type: text/plain
I0327 15:44:08.812] date: Tue, 27 Mar 2018 15:44:08 GMT
I0327 15:44:08.812] server: envoy
I0327 15:44:08.812] 
I0327 15:44:08.812] upstream connect error or disconnect/reset before headerscommand terminated with exit code 1
```

**Comment by cmluciano** [comment]
related flake https://github.com/istio/istio/issues/4420

**Pull request merged**